linguatec_lexicon/management/commands/importvariation.py

```diff
@@ -172,7 +172,7 @@ def retrieve_word(self, row_number, term_raw):
             # 3) trigam similarity (only as suggestion)
             message = 'Word "{}" not found in the database.'.format(term)
             suggestions = None
-            qs = Word.objects.search(term)[:4]
+            qs = Word.objects.search(term,None)[:4]
```

Comment:
Maybe could we use the selected lexicon to generate suggestions?

Reply:
yes, we could change this, but the principal problem about this is, the way the models is done we would have to change first the unique value in term in models.Word before because otherwise it can cause an error when importing variations if it finds two words with the same term. So I suggest doing this change in another branch along with the models.Word,term.

Reply:
I believe that that won't be a problem here because `.search()` filters words and doesn't expect to have a single result. In fact, including the proper lexicon to perform the search will reduce the number of results (not increase it). So my proposal is to replace line 175 with something like:
```python
            qs = Word.objects.search(term, lexicon)[:4]
```

That couldn't be done yet because `importvariation` command doesn't require this parameter. So, you're right... we should made this change later on another issue.

Anyway, I'm annoyed with the `Word.objects.search()` method definition. If lexicon parameter is optional (e.g. accepts None as value), we can replace the method definition by something like:
```python
    # linguatec_lexicon/models.py
    def search(self, query, lexicon=None):
```
And then, line line 175 with something like:
```python
            qs = Word.objects.search(term)[:4]
```

What do you think?

Reply:
I think you are right, it will be a better and more accurate option, to set the default value of the lexicon as None in the method definition, allowing to use it without writting the variable None when you don't want to search in a specific lexicon but in all of them. Which is probably more intuitive.